utf8.c

```diff
@@ -1619,7 +1619,6 @@ The caller, of course, is responsible for freeing any returned AV.
 =for apidoc Amnh||UTF8_ALLOW_LONG
 =for apidoc Amnh||UTF8_ALLOW_NON_CONTINUATION
 =for apidoc Amnh||UTF8_ALLOW_OVERFLOW
-=for apidoc Amnh||UTF8_ALLOW_PERL_EXTENDED
```

Comment:
Is there any possibility that someone on CPAN may be relying upon `UTF8_ALLOW_PERL_EXTENDED`?  If not, then this looks good.